src/docs/development/ui/assets-and-images.md

```diff
@@ -340,6 +340,13 @@ NSString* path = [[NSBundle mainBundle] pathForResource:key ofType:nil];
 For a more complete example, see the implementation of the
 Flutter [video_player plugin][].
```

Comment:
Don't you want to talk about your plugin in this section too for density resolution?

Reply:
done